docker/images/n8n/README.md

```diff
@@ -1,111 +1,108 @@
 
-> **WARNING**: This is only meant for local development and testing. Should not be used in production!
+If this data can't be found at startup n8n automatically creates a new key and any existing credentials can no longer be decrypted.
 
-To be able to use webhooks which all triggers of external services like Github
-rely on n8n has to be reachable from the web. To make that easy n8n has a
-special tunnel service (uses this code: [https://github.com/n8n-io/localtunnel](https://github.com/n8n-io/localtunnel)) which redirects requests from our servers to your local
-n8n instance.
+## Start n8n with tunnel
+
+> **WARNING**: This is only meant for local development and testing and should **NOT** be used in production!
+
+n8n must be reachable from the internet to make use of webhooks - essential for triggering workflows from external web-based services such as GitHub. To make this easier, n8n has a special tunnel service which redirects requests from our servers to your local n8n instance. You can inspect the code running this service here: [https://github.com/n8n-io/localtunnel](https://github.com/n8n-io/localtunnel)
 
 To use it simply start n8n with `--tunnel`
 
 ```bash
+docker volume create n8n_data
+
 docker run -it --rm \
  --name n8n \
  -p 5678:5678 \
- -v ~/.n8n:/home/node/.n8n \
+ -v n8n_data:/home/node/.n8n \
  docker.n8n.io/n8nio/n8n \
  start --tunnel
 ```
 
-## Persist data
-
-The workflow data gets by default saved in an SQLite database in the user
-folder (`/home/node/.n8n`). That folder also additionally contains the
-settings like webhook URL and encryption key.
-Note that the folder needs to be writable by user with UID/GID 1000.
-
-```bash
-docker run -it --rm \
- --name n8n \
- -p 5678:5678 \
- -v ~/.n8n:/home/node/.n8n \
- docker.n8n.io/n8nio/n8n
-```
-
-### Start with other Database
+## Use with PostgreSQL
 
-By default n8n uses SQLite to save credentials, past executions and workflows.
-n8n however also supports PostgresDB.
+By default, n8n uses SQLite to save credentials, past executions and workflows. However, n8n also supports using PostgreSQL.
 
-It is important to still persist the data in the `/home/node/.n8n` folder. The reason
-is that it contains n8n user data. That is the name of the webhook
-(in case) the n8n tunnel gets used and even more important the encryption key
-for the credentials. If none gets found n8n creates automatically one on
-startup. In case credentials are already saved with a different encryption key
-it can not be used anymore as encrypting it is not possible anymore.
+Note that even when using a different database, it is still important to
+persist the `/home/node/.n8n` folder, which also contains essential n8n
+user data including the encryption key for the credentials.
 
-#### Use with PostgresDB
-
-Replace the following placeholders with the actual data:
+In the following commands, replace these placeholders with the actual data:
 
 - POSTGRES_DATABASE
```

Comment:
This might be a strange question, but do you think we should include the angle brackets in the listing here? Not a big deal either way, just wondering which is easier to interpret.

Reply:
Yeah, I dunno. I am not a big fan of using brackets to denote user data, but my usual workaround is to define envars, which would be weirdly circular here. I don't think this list is particularly effective though, perhaps we should just describe the action?

Reply:
Yeah, I think that works. I'm a bit curious if they're all required (specifically the default schema is probably fine for most people), but I don't think we necessarily need to worry about that in this doc.